content/gettingstarted/index.adoc

```diff
@@ -48,6 +48,8 @@ icon:bullhorn[size=1x]{nbsp} This script requires `envsubst` installed and in yo
 curl -L http://git.io/getLatestKiali | bash
 ----
 
+By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKiali[the comments at the top of the install script].
```

Comment:
"By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali."

This is a bit wordy, we can probably trim it down to:

"By default, the installation script will prompt you to enter a username and passphrase  that you want users to enter to log in to Kiali."